Lib/test/test_struct.py

```diff
@@ -774,6 +774,10 @@ def test_error_propagation(fmt_str):
         test_error_propagation('N')
         test_error_propagation('n')
 
+    def test_repr(self):
+        s = struct.Struct('=i2H')
+        self.assertEqual(repr(s), f'Struct({s.format!r})')
```

Comment:
I think this could be updated to the following for clarity
```suggestion
        # s.format calls a read-only data descriptor provided by struct.Struct, and not
        # the similarly named str.format() method
        self.assertEqual(repr(s), f'Struct({repr(s.format)})')
```

since the `.format()` _**method**_ is frequently used in string formatting as well (e.g. https://peps.python.org/pep-3101/, https://peps.python.org/pep-0498/), and someone skimming the tests (such as me) might mix up the two at first glance and wonder why something like
```
f'Struct({repr(s)})'
# or
f'Struct({s!r})')
```
wasn't used instead.

Reply:
I don't think that comment is necessary. I would prefer to simply rename `s` to `struct_` or anything else, so that `s` doesn't look like a `str`.

Reply:
I dont think that comment is necessary too. Word `format` is used a lot in this file, so it is unlikely be confused with `str.format`.

`s` is common name for structs in this test suite. So I think `s` would be more consistent than `struct_`.